examples/image/base_image.py

```diff
@@ -21,7 +22,7 @@ async def t1(data: str = "hello") -> str:
 
 
 if __name__ == "__main__":
-    flyte.init_from_config("../../config.yaml")
+    flyte.init_from_config(log_level=logging.INFO)
```

Comment:
Honestly, I do not love this magic.
How about
```python
flyte.config.auto() or
cfg_file = flyte.config.find_config()
flyte.init_from_config(cfg_file)
```

Reply:
I like the idea of `.flyte` we could use it store local cache data and other data for runs. But, then we dont just want up - we look at current folder or provide path

Reply:
> But, then we dont just want up - we look at current folder or provide path

At the same time I think the current behavior is kinda janky imo. You have to be aware of the current working directory and the relative path of the flyte script to the config file to get it to work properly... just look at our examples repo. you have a bunch of `../../../../config.yaml` paths that look pretty ugly.

Reply:
> flyte.config.auto() or
> cfg_file = flyte.config.find_config()
> flyte.init_from_config(cfg_file)

This isn't gonna do anything. `flyte.init_from_config` already calls `config.auto()` with the current behavior.

Reply:
i know, what i was saying was modify something, but you are right, dont want the magic to be behind the scenes

Reply:
I like it. I actually like more of the git helpers, like `flyte.git.config_from_root` or `flyte.git.version_from_sha`, `flyte.git.path`